tests/run-make/link-arg/rmake.rs

```diff
@@ -0,0 +1,19 @@
+// In 2016, the rustc flag "-C link-arg" was introduced - it can be repeatedly used
+// to add single arguments to the linker. This test passes 2 arguments to the linker using it,
+// then checks that the compiler's output contains the arguments passed to it.
+// This ensures that the compiler successfully parses this flag.
+// See https://github.com/rust-lang/rust/pull/36574
+
+use run_make_support::rustc;
+
+fn main() {
+    // The original Makefile test did not check if rustc succeeded or failed.
```

Comment:
Suggestion: the original Makefile doesn't really matter here, but instead we should explain *why* this is unchecked, i.e.

> We are only checking for the output of `--print=link-args`, exit status does not matter here.